### PR TITLE
Use setup.cfg

### PR DESCRIPTION
**Pull request by riedgar-ms**
Move the data embedded in `setup.py` into the `setup.cfg` file. This stops us mixing code and data, and will allow the consistency checking script for requirements to be extended to cover `setup.cfg` as well

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,43 @@
+[metadata]
+name = fairlearn
+version = attr: fairlearn.__version__
+author = Miroslav Dudik
```
Comment (romanlutz) [inline setup.cfg]: Time to adjust that? I think this can also have authors instead of singular author?

File: setup.cfg [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+author_email = mdudik@gmail.com
+description = Algorithms for fair classification
```
Comment (romanlutz) [inline setup.cfg]: also should be adjusted...

**Comment by romanlutz** [comment]
I hope this won't break other things. I think a lot of tools are used to checking setup.py

**Comment by riedgar-ms** [comment]
@romanlutz  which tools check `setup.py` as well?

**Comment by riedgar-ms** [comment]
Closing for now. I still think this might be a good idea, but we need to sort out what's happening with both this and requirements.txt

**Pull request closed without merging**